aws/hhvm1/init-update-repos.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -ex
+
+cleanup() {
+  umount /mnt/dl.hhvm.com
+}
+
+INSTANCE_ID=$(curl --retry 5 http://169.254.169.254/latest/meta-data/instance-id)
+
+# We keep a shared XFS volume as the source of truth to avoid having to
+# redownload from S3 every day, which can take a *really* long time.
+VOLUME_ID="vol-096d2c45d13d3a865"
+
+try_really_hard aws ec2 attach-volume \
```

Comment:
descriptive function name

Reply:
Now I'm not sure if you're praising the descriptiveness of this one or asking for a more descriptive one :) Can't really think of a much better name, maybe `retry_or_fail` or `retry_then_fail`...?